tests/unit/animations/cssTransitions.ts

```diff
@@ -0,0 +1,84 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import dispatchSyntheticEvent from '../../support/dispatchSyntheticEvent';
+import cssTransistions from '../../../src/animations/cssTransitions';
+
+registerSuite({
+	name: 'animations/cssTransistions',
+
+	'enter': {
+		'transitionend'(this: any) {
+			const dfd = this.async();
+			const div = document.createElement('div');
+			cssTransistions.enter(div, {}, 'foo');
+			assert.lengthOf(div.classList, 1);
+			assert.isTrue(div.classList.contains('foo'));
+
+			setInterval(dfd.callback(() => {
```

Comment:
why is this `setInterval` over `setTimeout`? I can't see any teardown of the interval either?